src/constants.ts

```diff
@@ -28,5 +28,13 @@ export const CHAIN_IDS = {
   AVALANCHE: 43114,
 };
 
-export const ZEROEX_CONTRACT_ADDRESS =
-  '0xdef1c0ded9bec7f1a1670819833240f027b25eff';
+export const EXCHANGE_PROXY_ADDRESSES = {
+  1: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
+  5: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
+  10: '0xdef1abe32c034e558cdd535791643c58a13acc10',
+  56: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
+  137: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
+  250: '0xdef189deaef76e379df891899eb5a00a94cbc250',
+  42220: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
+  43114: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
+};
```

Comment:
might add a function facade to offer little bit of validation around it rather than accessing array index directly

```suggestion
export const EXCHANGE_PROXY_ADDRESSES = {
  1: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
  5: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
  10: '0xdef1abe32c034e558cdd535791643c58a13acc10',
  56: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
  137: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
  250: '0xdef189deaef76e379df891899eb5a00a94cbc250',
  42220: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
  43114: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
};

const getExchangeProxyAddress = (chainId: number) => {
  const address = EXCHANGE_PROXY_ADDRESSES[chainId]
  if (!address) { 
    throw new Error(`Chain ${chainId} not yet supported by 0x API`)
   }
}
```